docs/setup.md

```diff
@@ -100,7 +100,120 @@ in their associated documentation (chapter "Components").
 
 If you are not using a public ledger, you will need to run your own local one.
 
-...run conodes, make ledger, get bc file, set BC env variable in variables.sh, etc.
+#### Run the nodes
+
+Build the conode binary and use the `run_nodes.sh` script in order to get a set
+of conodes up and running:
+
+```bash
+cd ledger/conode
+go build
+./run_nodes.sh -d tmp -v 2
```

Comment:
I'm not sure what the point of `tmp` is. I used no -d arg, but if you use, it, please make it something that says, "I'm important, don't delete me", like `conode_data`.

Reply:
tmp is perhaps not the best choice, its an old habit when I use run_nodes.sh to quickly spin up a temporary cothority for development purposes. But still, I find it much more manageable to separate the cothority files and folders from the conode code. Its also easier to see and understand what the run_nodes.sh script do, and its also easier to exclude this single folder from git.